src/aleph/http/server.clj

```diff
@@ -111,8 +111,8 @@
            (.printStackTrace e (java.io.PrintWriter. w))
            (str w))})
 
-(let [[server-name connection-name date-name]
-      (map #(HttpHeaders/newEntity %) ["Server" "Connection" "Date"])
+(let [[server-name connection-name date-name content-type]
+      (map #(HttpHeaders/newEntity %) ["Server" "Connection" "Date" "Content-Type"])
 
       [server-value keep-alive-value close-value]
       (map #(HttpHeaders/newEntity %) ["Aleph/0.6.0" "Keep-Alive" "Close"])]
```

Comment:
...why do we have two `map` calls?

???

Reply:
I will merge them but I guess it was for readability reasons only.

Reply:
0fef219efcf69a0a01b582619c994004159c95d7

Reply:
Actually, I'm going to revert my changes... I understand why it has been done that way.

The first `map` is for the __headers keys__.
The second `map` represents __headers values__.

It's nice to have a quick look at the headers we are manipulating when sending the response instead of a medley.